pkg/kubectl/cmd/drain_test.go

```diff
@@ -32,12 +32,13 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
+	"k8s.io/api/core/v1"
```

Comment:
I think we'd better using `api.Node`(internal type) instead of `v1.Node` (serialized type) here. because internal type could change and our unit test should perceive that. @mengqiy wdyt ?